palace/utils/geodata.cpp

```diff
@@ -682,31 +682,38 @@ void GetAxisAlignedBoundingBox(const mfem::ParMesh &mesh, const mfem::Array<int>
 
 double BoundingBox::Area() const
 {
-  return 4.0 *
-         CVector3dMap(normals[0].data()).cross(CVector3dMap(normals[1].data())).norm();
+  return 4.0 * CVector3dMap(axes[0].data()).cross(CVector3dMap(axes[1].data())).norm();
 }
 
 double BoundingBox::Volume() const
 {
-  return planar ? 0.0 : 2.0 * CVector3dMap(normals[2].data()).norm() * Area();
+  return planar ? 0.0 : 2.0 * CVector3dMap(axes[2].data()).norm() * Area();
+}
+
+std::array<std::array<double, 3>, 3> BoundingBox::Normals() const
+{
+  std::array<std::array<double, 3>, 3> normals = {axes[0], axes[1], axes[2]};
+  Vector3dMap(normals[0].data()).normalize(), Vector3dMap(normals[1].data()).normalize(),
+      Vector3dMap(normals[2].data()).normalize();
```

Comment:
Not a fan of using `,` generally as it can be a little surprising sometimes. `;` or a range loop is preferable. 
```
  for (auto &x : normals)
  {
    Vector3dMap(x.data()).normalize();
  }
  ```

Reply:
Oh whoops, yeah these were supposed to be semicolons! Good catch.

Reply:
See [3538db1](https://github.com/awslabs/palace/pull/248/commits/3538db1170eaea54d0ebf6c732915f403c0afabd).